lib/arena.rb

```diff
@@ -1,2 +1,65 @@
+require "pry"
+
 class Arena
-end
+  attr_accessor(:name, :gladiators, :vote, :losing_gladiators)
+
+  def initialize(name)
+      @name = name.capitalize
+      @gladiators = []
+  end
+
+  def add_gladiator(gladiator)
+    return if @gladiators.length >= 2
+    @gladiators << gladiator
+  end
+
+  def remove_gladiator_name
+    @gladiators.reject! { |gladiator| gladiator.name == "Hercules"} #.name refers to attribute of @gladiators, "gladiator" can say anything, it's like "i".
+  end #end remove_gladiator_name
+
+  def fight
+    return if @gladiators.length <= 1
+
+    @losing_gladiators = []
+    puts "#{@gladiators.first.weapon} and #{@gladiators.last.weapon}"
+    if @gladiators.any? { |gladiator| gladiator.name == "Maximus"}
+    puts "Maximus won!"
+    return
+
+    elsif @gladiators.first.weapon === @gladiators.last.weapon
+        @gladiators.clear
+
+    elsif @gladiators.first.weapon == "Spear" && @gladiators.last.weapon == "Trident"
+      puts "Hi"
+        @losing_gladiators << @gladiators[0]
+        #@gladiators.shift
+
+    elsif @gladiators.first.weapon == "Club" && @gladiators.last.weapon == "Spear"
+        @gladiators.shift
```

Comment:
Why do you switch logic for losing/winning gladiators?  On line 34 you added the loser to `@losing_gladiators`, now we are removing the loser from `@gladiators`.  In order to allow for a vote, don't we need both gladiators in the Arena at the end?